File: ios/Sources/Mekuri/MekuriPageMode.swift
/// How a page is being drawn, read from `\.mekuriPageMode`. A `.turning`
/// page is rasterized and must not depend on live playback or interaction.
public enum MekuriPageMode: Equatable, Sendable {
    /// The page is settled on screen; interaction and playback run.
    case live
    /// The page is a face of the leaf being turned and is drawn through the
    /// fold shader from a flattened copy.
    case turning
}

File: android/mekuri/src/main/kotlin/app/mekuri/MekuriPageMode.kt
package app.mekuri

/**
 * How a page is being drawn. A [Turning] page is drawn through the fold shader
 * and must not depend on live playback or interaction.
 */
public enum class MekuriPageMode {
    /** The page is settled on screen; interaction and playback run. */
    Live,

    /** The page is a face of the leaf being turned. */
    Turning,
}
